src/Npgsql/PublicAPI.Shipped.txt

```diff
@@ -1907,3 +1907,7 @@ override NpgsqlTypes.NpgsqlRange<T>.RangeTypeConverter.CanConvertFrom(System.Com
 override NpgsqlTypes.NpgsqlRange<T>.RangeTypeConverter.CanConvertTo(System.ComponentModel.ITypeDescriptorContext? context, System.Type? destinationType) -> bool
 override NpgsqlTypes.NpgsqlRange<T>.RangeTypeConverter.ConvertFrom(System.ComponentModel.ITypeDescriptorContext? context, System.Globalization.CultureInfo? culture, object! value) -> object?
 override NpgsqlTypes.NpgsqlRange<T>.RangeTypeConverter.ConvertTo(System.ComponentModel.ITypeDescriptorContext? context, System.Globalization.CultureInfo? culture, object? value, System.Type! destinationType) -> object?
+Npgsql.NpgsqlConnectionStringBuilder.EnableDiscardSequences.get -> bool
+Npgsql.NpgsqlConnectionStringBuilder.EnableDiscardSequences.set -> void
+Npgsql.NpgsqlConnectionStringBuilder.EnableDiscardTemp.get -> bool
```

Comment:
Temp tables should be discarded irrespective because the application may see unexpected errors.